src/test/java/org/cactoos/http/HtAutoRedirectTest.java

```diff
@@ -53,7 +56,11 @@ public void redirectsRequestAutomatically() throws Exception {
                             new JoinedText(
                                 "\r\n",
                                 "HTTP/1.1 301",
-                                String.format("Location: %s", home)
+                                new UncheckedText(
```

Comment:
@alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `Exception`

Reply:
Fixed